### DISCUSSION TITLE
[Feature] Image editing on Android

### DISCUSSION THREAD

**Discussion by benjamindaines** [immich-app/immich#9213] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Immich is very close to being able to replace the default gallery app on my S22.  The biggest feature that is missing is the ability to edit photos; even something as simple as cropping.  I also don't see anyway to "open with..." in another app (for example, the Samsung Gallery app that does support editing and some other features such as OCR).  

If an editing feature could be added to Immich, that would be great.  If Immich could somehow make use of the already complete editing interface that Samsung (or other vendor / Google themselves) provides, that would perhaps be even better as it would support all the advertised feature of the phone and save Immich some development hours. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
Duplicate of https://github.com/immich-app/immich/issues/768
